src/NATS.Client.KeyValueStore/NatsKVContext.cs

```diff
@@ -108,9 +108,13 @@ public async ValueTask<INatsKVStore> CreateStoreAsync(NatsKVConfig config, Cance
             // MirrorDirect =
             // Mirror =
             Retention = StreamConfigRetention.Limits, // from ADR-8
-            DuplicateWindow = TimeSpan.FromMinutes(2), // 120_000_000_000ns, from ADR-8
+            DuplicateWindow = config.MaxAge != default ? config.MaxAge : TimeSpan.FromMinutes(2), // 120_000_000_000ns, from ADR-8
```

Comment:
my fault didn't check setting duplicate-window to max-age here should've been removed
```diff
- DuplicateWindow = config.MaxAge != default ? config.MaxAge : TimeSpan.FromMinutes(2), // 120_000_000_000ns, from ADR-8
+ DuplicateWindow = TimeSpan.FromMinutes(2), // 120_000_000_000ns, from ADR-8
```

cc @scottf

Background: When setting max age default window is always set equals to max age. e.g. if we set max-age=30 minute default windows should still stay at 2 minutes.